frontend/packages/knative-plugin/src/components/test-function/modal-components/RequestBody.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { FormikProps, FormikValues } from 'formik/dist/types';
+import { YAMLEditorField } from '@console/shared/src';
+import { InvokeFormat } from '../../../types';
+
+const RequestBody: React.FC<FormikProps<FormikValues>> = ({ values }) => {
+  const {
+    request: {
+      headers: {
+        [InvokeFormat.CloudEvent]: { ContentType: CloudEventContentType },
+        [InvokeFormat.HTTP]: { ContentType: HTTPContentType },
+      },
```

Comment:
see above. We should use fields for specific forms and then one for the header-array. And we must not save the values separately for cloudevent and http.